src/com/palmergames/bukkit/config/ConfigNodes.java

```diff
@@ -1528,7 +1528,7 @@ public enum ConfigNodes {
 			"# A full list of proper names can be found here https://hub.spigotmc.org/javadocs/spigot/org/bukkit/Material.html "),
 	PROT_SWITCH_MAT(
 			"protection.switch_ids",
-			"CHEST,SHULKER_BOXES,TRAPPED_CHEST,FURNACE,BLAST_FURNACE,DISPENSER,HOPPER,DROPPER,JUKEBOX,SMOKER,COMPOSTER,BELL,BARREL,BREWING_STAND,LEVER,PRESSURE_PLATES,BUTTONS,WOOD_DOORS,FENCE_GATES,TRAPDOORS,MINECARTS,LODESTONE,RESPAWN_ANCHOR,TARGET,OAK_CHEST_BOAT",

```

Comment:
Yes, but if this is something that wouldn't be accepted I wouldn't want them to put any more effort.

@GroobleDierne I will commend you though, it is coded well and you've been pretty thorough without receiving any direction from the team as of yet.

Reply:
> Having a config migration that automatically removes containers from existing configs might be nice in order to prevent people from thinking they can configure containers there

I saw the config-migration.json but wasn't sure how to use it so I preferred not to touch it